include/MixerChannelView.h

```diff
@@ -32,113 +38,107 @@
 #include "PixmapButton.h"
 #include "SendButtonIndicator.h"
 
-#include <QGraphicsView>
-#include <QLabel>
-#include <QLineEdit>
-#include <QPixmap>
-#include <QWidget>
-
-namespace lmms
-{
-    class MixerChannel;
+namespace lmms {
+class MixerChannel;
 }
 
-namespace lmms::gui
+namespace lmms::gui {
```

Comment:
Wouldn't it be better to just use ::gui, and wrap the lmms namespace all the way to down?

Reply:
Not exactly sure what you mean here, but I find saying `lmms::gui` once is easier rather than nesting `lmms` and `gui`.

Reply:
I meant nesting but it's still fine if you don't nest